src/Laravel/DiscoveryController.php

```diff
@@ -13,8 +14,10 @@ class DiscoveryController
      */
     public function __invoke(Request $request)
     {
+        URL::forceScheme('https'); // for route() calls below
+
         $response = [
-            'issuer' => url('/'),
+            'issuer' => 'https://' . $_SERVER['HTTP_HOST'],
```

Comment:
@jeremy379 This broke my unit test which was calling it without a HTTP_HOST. Maybe a good idea to fallback on both to route('/'), so also for the Id & Access tokens?

Reply:
Hello
I'm sorry for these issue

Request isn't directly available here, so I choose to provide option on how this should be filled.

Here is a fix MR: https://github.com/jeremy379/laravel-openid-connect/pull/33